modules/fts/new_topics/fts-search-request-details.adoc

```diff
@@ -0,0 +1,978 @@
+    "query": {
+        "boost": 1,
+        "query": "water"
+    },
+    "facets": {
+        "abv": {
+            "size": 5,
+            "field": "abv",
+            "numeric_ranges": [
+                {
+                    "name": "high",
+                    "min": 7
+                },
+                {
+                    "name": "low",
+                    "max": 7
+                }
+             ]
+        }
+    }
+}'
+----
++
+Results:
++
+[source,json]
+----
+facets": {
+    "abv": {
+        "field": "abv",
+        "total": 70,
+        "missing": 21,
+        "other": 0,
+        "numeric_ranges": [
+            {
+                "name": "high",
+                "min": 7,
+                "count": 13
+            },
+            {
+                "name": "low",
+                "max": 7,
+                "count": 57
+            }
+        ]
+    }
+}
+----
+
+[#date-range-facet]
+=== Date Range Facet
+
+The Date Range facet is same as numeric facet, but on dates instead of numbers.
+
+Full text search and Bleve expect dates to be in the format specified by https://www.ietf.org/rfc/rfc3339.txt[RFC-3339^], which is a specific profile of ISO-8601 that is more restrictive.
+
+==== Example
+
+* Date Range Facet - computes facet on the ‘updated’ field that has 2 values old and new.
++
+----
+curl -XPOST -H "Content-Type: application/json" -uAdministrator:asdasd http://<node>:8094/api/index/bix/query -d '{
+"ctl": {"timeout": 0},
+"from": 0,
+"size": 0,
+"query": {
+            "field": "country",
+            "term": "united"
+},
+        "facets": {
+        "types": {
+        "size": 10,
+        "field": "updated",
+        "date_ranges": [
+        {
+        "name": "old",
+        "end": "2010-08-01"
+        },
+        {
+        "name": "new",
+        "start": "2010-08-01"
+        }
+]
+}
+}
+}'
+----
++
+Results
++
+[source,json]
+----
+ "facets": {
+             "types": {
+                "field": "updated",
+                "total": 954,
+                "missing": 0,
+                "other": 0,
+             "date_ranges": [
+              {
+                "name": "old",
+                "end": "2010-08-01T00:00:00Z",
+                "count": 934
+              },
+              {
+                "name": "new",
+                "start": "2010-08-01T00:00:00Z",
+                "count": 20
+              }
+               ]
+             }
+           }
+----
+
+[#Collections]
+== Collections
+
+Collections field let’s the user specify an optional list of collection names. This would help the users scope their search request to only those specified collections within a multi-collection index. This becomes useful with multi-collection indexes as it can speed up searches, as well as the user can manage the Role Based Access Control more granularly with this option. Ie the search user only needs permissions for the requested collections and not for every other collection indexed within the index.
```

Comment:
```suggestion
Collections field let’s the user specify an optional list of collection names. This would help the users scope their search request to only those specified collections within a multi-collection index. This becomes useful with multi-collection indexes as it can speed up searches, as well as allowing the user to manage the Role Based Access Control with more granularity. 
I.e.  the search user only needs permissions for the requested collections and not for every other collection indexed within the index.
```